__tests__/server.test.js

```diff
@@ -240,3 +239,41 @@ describe('GET /api/users', () => {
             })
     })
 });
+describe('GET /api/articles/:article_id/comments', () => {
+    test('tests the connection to the GET /api/articles/:article_id/comments endpoint where comments exist', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body: { comments } }) => {
+            const erroneousIds = comments.filter(comment => {
+                comment.article_id !== 1
+            })
+            expect(comments.length).toBe(11);
+            expect(erroneousIds.length).toBe(0);
+        })
+    })
```

Comment:
I'd want to see a test checking the comments that come back have the right structure aka the right keys